### DISCUSSION TITLE
Must the profile picture be square?

### DISCUSSION THREAD

**Discussion by Rod-in-NM** [community/community#49651] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to add a profile picture for my company's new GitHub organization. However, the logo I'm working with is a rectangle, not a square. When I uploaded it, the website insisted upon my cropping it, or go with a default cropping, which I did, which was the largest it would allow, but is still chopping of text on the left and right sides. The imagine is 500 x 338 pixels and is 682 KB in size.

**Comment by Abinashbunty** [comment, marked as answer]
Yes, it is indeed required to have a square image. Perhaps GitHub does this to maintain uniformity. 

But you can use a tool like [this](https://squaremyimage.com/) to generate square image of your rectangle one. 😄  Hope it answers your query.

**Reply by Abinashbunty** [reply]
> Yes, it is indeed required to have a square image. Perhaps GitHub does this to maintain uniformity.
> 
> But you can use a tool like [this](https://squaremyimage.com/) to generate square image of your rectangle one. 😄 Hope it answers your query.

You can mark it as answer if it helped you. Thanks in advance. 😄

**Reply by zuck30** [reply]
perfect.. had similar issue too.

**Comment by ashmeet07** [comment]
## 🖼️ GitHub Profile Picture Requirement

**Yes, GitHub profile pictures (avatars) for both user accounts and organizations must be square.**

GitHub enforces this square aspect ratio to ensure **uniformity** and proper scaling, as all avatars are ultimately displayed within a **circular mask**. When you upload a rectangular image, the cropping tool forces you to select only a square area, which results in your text being cut off.

---

### 🛠️ Solution: How to Fit a Rectangular Logo

The best way to display your full rectangular logo (like your 500x338 example) without cropping is to place it onto a larger square canvas using any image editor.

#### Step-by-Step Action Plan:

1.  **Create a Square Canvas:** Start a new image file with square dimensions, such as **500x500 pixels** (or a larger size like 1000x1000 for better quality).
2.  **Center Your Logo:** Paste your original rectangular logo (500x338) directly into the center of the new square canvas.
3.  **Fill the Edges:** Fill the empty space above and below your logo with a **solid background color** that matches your branding (e.g., white, black, or your main logo color).
4.  **Save and Upload:** Save this new, squared image (e.g., 500x500). When you upload it to GitHub, the cropping tool will select the entire canvas, and your full logo will appear perfectly centered within the circular avatar.

**Reply by zuck30** [reply]
[https://squaremyimage.com/](url) that site can crop your logo to square for github profile picture.
